server/monitor/src/main/java/org/apache/accumulo/monitor/rest/resources/GarbageCollectorResource.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.monitor.rest.resources;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.apache.accumulo.core.gc.thrift.GCStatus;
+import org.apache.accumulo.monitor.Monitor;
+import org.apache.accumulo.monitor.rest.api.GarbageCollection;
+import org.apache.accumulo.monitor.rest.api.GarbageCollectorCycle;
+import org.apache.accumulo.monitor.rest.api.GarbageCollectorStatus;
+
+/**
+ * GarbageCollector metrics
+ */
+@Path("/gc")
+@Produces({MediaType.APPLICATION_JSON, MediaType.APPLICATION_XML})
+public class GarbageCollectorResource {
+
+  @GET
```

Comment:
I tried moving `@GET` to class level and it says _The annotation @GET is disallowed for this location_, I didn't find a way to do it on class level during my Google search, do you have an idea of how this might be done?